### PR TITLE
A lot of changes and a more reponsible engineer would break these out into concise PRs but I'm not that guy and write titles that are way too long

### PR DESCRIPTION
**Pull request by Jonpro03**

### CODE REVIEW

File: Obsidian/Client.cs [hunk 1]
```diff
@@ -520,7 +520,6 @@ internal async Task ConnectAsync()
         Logger.LogDebug("Sent Login success to user {Username} {UUID}", Player.Username, Player.Uuid);
 
         this.State = ClientState.Play;
-        await Player.LoadAsync();
```
Comment (Tides) [inline Obsidian/Client.cs]: Why was this moved :eyes:

Comment (Jonpro03) [reply Obsidian/Client.cs]: I was trying to fix a bug that when a player logs in but they are nowhere near spawn, we fail sending them chunks and spawn them anyway (into the void). This is still broken.

But, I noticed that we get the ClientInformationPacket before Client.ConnectAsync is called. So when we call SendInfoAsync() from ClientInformationPacket, the player is not loaded yet and we send bad data.
Admittedly, LoadAsync shouldn't be in a packet handler. It was half-baked and I didn't mean to leave it in here. I quit working on that bug so I'll do this properly when I fix that bug

Comment (Tides) [reply Obsidian/Client.cs]: I already fixed that bug ^ ^

**Review by Craftplacer — DISMISSED** [review]

File: Obsidian/Client.cs (hunk 1 above)
Comment (Jonpro03) [reply Obsidian/Client.cs]: still broken, I'm afraid

Comment (Tides) [reply Obsidian/Client.cs]: Yeah I confused it for another bug but I think I know how to fix this one

Comment (Jonpro03) [reply Obsidian/Client.cs]: There is a return value on send chunks that we need to utilize

**Review by Tides — APPROVED** [review]

**Review by Naamloos — APPROVED** [review]

**Comment by Naamloos** [comment]
LGTM merging
